test/objects.test.js

```diff
@@ -396,10 +415,10 @@ describe('objects', function() {
 
           var dog = null,
               newBreed = "";
-
+          console.log('body2',body);
```

Comment:
Sorry remove it, please.
I have found that, but git commit --amend seems only works in local.